packages/logstash/data_stream/node_stats/fields/package-fields.yml

```
@@ -84,11 +84,29 @@
           type: group
           fields:
             - name: cpuacct
-              type: object
+              type: group
               fields:
                 - name: usage_nanos
                   type: alias
                   path: logstash.node.stats.os.cgroup.cpuacct.usage_nanos
+            - name: cpu
+              type: group
+              fields:
+                - name: cfs_quota_micros
+                  path: logstash.node.stats.os.cgroup.cpu.cfs_quota_micros
+                  type: alias
+                - name: stat
+                  type: group
+                  fields:
+                    - name: number_of_elapsed_periods
+                      path: logstash.node.stats.os.cgroup.cpu.stat.number_of_elapsed_periods
+                      type: alias
+                    - name: number_of_times_throttled
+                      path: logstash.node.stats.os.cgroup.cpu.stat.number_of_times_throttled
+                      type: alias
+                    - name: time_throttled_nanos
+                      path: logstash.node.stats.os.cgroup.cpu.stat.time_throttled_nanos
+                      type: alias
```

NIt: To me it makes sense to read it's a type alias before reading the path it's aliased to.

```suggestion
                  type: alias
                  path: logstash.node.stats.os.cgroup.cpu.cfs_quota_micros
                - name: stat
                  type: group
                  fields:
                    - name: number_of_elapsed_periods
                      type: alias
                      path: logstash.node.stats.os.cgroup.cpu.stat.number_of_elapsed_periods
                    - name: number_of_times_throttled
                      type: alias
                      path: logstash.node.stats.os.cgroup.cpu.stat.number_of_times_throttled
                    - name: time_throttled_nanos
                      type: alias
                      path: logstash.node.stats.os.cgroup.cpu.stat.time_throttled_nanos
```